### PR TITLE
doc: wg: add core working group charter

### PR DESCRIPTION
**Pull request by bradjc**
### Pull Request Overview

This pull request adds a charter for the core working group. This is a bit of a formalization of the existing CoreTeam.md document.

### Testing Strategy

n/a

### TODO or Help Wanted

Comments

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [x] Ran `make formatall`.

### CODE REVIEW

File: doc/wg/core.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- Amit Levy (chair), [alevy](https://github.com/alevy), Princeton University
+- Pat Pannuto, [ppannuto](https://github.com/ppannuto), UCSD
+- Johnathan Van Why, [jrvanwhy](https://github.com/jrvanwhy), Google
+
+## Membership
+
+The core working group membership is a subset of the people who have commit
+(pull request merge) permissions on the Tock repository. It is intended to be a
+smaller group that represents the major perspectives and issues, rather than a
+complete group. Contributors who are actively help develop Tock will be
```
Comment (jrvanwhy) [inline doc/wg/core.md]: "who are actively help" -> "who help"

File: doc/wg/core.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+To join the core team, a contributor must be nominated by an existing core team
+member. The nominator will open a pull request updating this document with the
+new core team member in the list above. That pull request will undergo the usual
+pull request review, and the member will be added to the core team if the pull
+request is merged.
+
+## Communication
+
+The group has a weekly teleconference call. All working group members are
+invited to   participate in the call. Other people may be invited to participate
```
Comment (jrvanwhy) [inline doc/wg/core.md]: There's some inconsistent whitespace here (three spaces in an row).

File: doc/wg/core.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  effect new code contributions will have on Tock's adherence to those
+  principles.
+- Understands the code style and structure of Tock and can help ensure a
+  reasonably consistent code base.
+- Understands Tock's various stakeholders and can judge how a change to Tock
+  might affect various users of Tock.
+
+A core team member is expected to:
+
+-Help review a percentage of new pull requests to the Tock code base. -Provide
```
Comment (alistair23) [inline doc/wg/core.md]: There should be a new line here.

**Comment by alevy** [comment]
If you're in @tock/core-wg, please comment, request changes, or approve

**Pull request merged**